### PR TITLE
removed ellipses and added spaces to properly format code quote

### LINKED ISSUE #1247: Missing <pre> in temp-release-notes
https://jujucharms.com/docs/devel/temp-release-notes

is missing some sort of <pre> stanza around these lines:
<p>Example bundle.yaml excerpt:
    ...
    mysql:
        charm: cs:trusty/mysql-53
        num_units: 1
        constraints: mem=4G
        bindings:
            server: database
            cluster: internal
    ...</p>

### PR DESCRIPTION
**Pull request by matthewhelmke**
Fixes #1247 
@juju/docs Ready for review

### CODE REVIEW

File: src/en/temp-release-notes.md [hunk 1]
```diff
@@ -585,15 +585,14 @@ when deploying individual charms. The bundle YAML can include a section
 called "bindings", defining the map of endpoint names to space names.
 
 Example bundle.yaml excerpt:
-    ...
+
```
Comment (evilnick) [inline src/en/temp-release-notes.md]: for the purposes of docs, it is best to code fence these so the highlighting (may) work:

```yaml
   
```

Comment (evilnick) [inline src/en/temp-release-notes.md]: but not indented - that was just to make it show up!

Comment (matthewhelmke) [inline src/en/temp-release-notes.md]: I'm happy to do that. I just followed most of the (currently rendered properly) other code boxes on the page. Shall I go through the whole thing?

Comment (evilnick) [inline src/en/temp-release-notes.md]: no, it's fine. they are only temporary after all - I just thought as you were fixing it, you might as well fix it nicely :)

**Comment by matthewhelmke** [comment]
They are better with the backticks and code type indicator...

**Comment by evilnick** [comment]
:cake: 
We can fancify it better when there is a proper release

**Pull request merged**